virtual_rainforest/models/animals/animal_cohorts.py

```diff
@@ -107,6 +113,10 @@ def __init__(
             self.mass,
             self.functional_group.prey_scaling,
         )
+        # TODO - In future this should be parameterised using a constants dataclass, but
+        # this hasn't yet been implemented for the animal model
+        self.decay_fraction_excrement: float = DECAY_FRACTION_EXCREMENT
+        """The fraction of excrement which decays before it gets consumed."""
```

Comment:
Do I understand correctly that this decay_fraction_excrement would then (later) be a function of our time-step size?

Reply:
Nope, I set it up so that this would be time step size independent. At the end of the time step all excrement in the decay pool is transferred to the litter pool. The excrement in the scavengable subpool never gets transferred to the litter and if nothing scavenges it it just builds up in that pool (which is obviously a little odd). 

The reason for this slightly odd division is that if we say set 50% of the excrement pool to decay at the end of every time step, shorter time steps would result in a reduced steady state excrement pool unless we did something clever to control for that. We might want to do something like that in the future, but I thought having two totally independent excrement subpools covering the two destinations (litter, scavenging) was easier for now